docs/dao-operations/dao-roles.md

```
@@ -163,6 +166,32 @@ To claim a Sync Guild role a member will need to claim two other roles:
+
+:::info[Latest budget]
+
+~3k per year
+
+:::
+
+_It's expected that a majority of the budget is allocated to completing the actual execution/implementation tasks._
+
+### Angry Dwarf - Keeper of the Treasury
+
+The [_Angry Dwarf ](https://app.hatsprotocol.xyz/trees/100/92?hatId=92.1.5.6) is responsible for creating quarterly accounting reports, ensuring cleric SOPs are followed, and implementing sensible portfolio management.
```

markdownlint-cli2 (0.22.0)

[warning] Spaces inside link text

(MD039, no-space-in-links)